kubernetes/samples/charts/traefik/README.md

```diff
@@ -4,102 +4,95 @@ This sample demonstrates how to install the Traefik ingress controller to provid
 load balancing for WebLogic clusters.
 
 ## Install the Traefik operator with a Helm chart
-The Traefik Helm chart is located in the official Helm project `charts` directory at https://github.com/helm/charts/tree/master/stable/traefik.
-The chart is in the default repository for Helm.
+This document is based on Traefik version 2.x with Helm chart at https://github.com/containous/traefik-helm-chart.
+For more information about Traefik, see https://docs.traefik.io/
 
 To install the Traefik operator in the `traefik` namespace with default settings:
 ```
-$ helm install --name traefik-operator --namespace traefik stable/traefik
+$ helm repo add traefik https://containous.github.io/traefik-helm-chart
+$ helm repo update
+$ helm install traefik-operator traefik/traefik --namespace traefik
 ```
-Or, with a given `values.yaml`:
+Or, with a given `values.yaml` For more detailed information, see https://github.com/containous/traefik-helm-chart/blob/master/traefik/values.yaml for more detail.
```

Comment:
this sentence does not make sense

Reply:
Reformatted the description
You can also install the Traefik operator, with a given `values.yaml`. For more detailed information, see https://github.com/containous/traefik-helm-chart/blob/master/traefik/values.yaml for more detail.

Reply:
Re formatted the description as follows ...
You can also install the Traefik operator, with a given `values.yaml`. For more detailed information, see [Traefik GitHub Project](https://github.com/containous/traefik-helm-chart/blob/master/traefik/values.yaml) for more detail.

Reply:
No, the idea here was to get rid of the phrase "for more detail" at the end of the sentence. It should be, "For more detailed information, see <something>.

Reply:
You can also install the Traefik operator, with a custom `values.yaml`. For more detailed information, see [Traefik GitHub Project](https://github.com/containous/traefik-helm-chart/blob/master/traefik/values.yaml).